.buildkite/react-native-cli-pipeline.yml

```diff
@@ -1,23 +1,81 @@
 steps:
 
   #
-  # Android builder
+  # CLI tests Maze Runner image
+  #
+  - label: ':docker: Build CLI test image'
+    key: 'cli-maze-image'
+    timeout_in_minutes: 30
+    plugins:
+      - docker-compose#v3.7.0:
+          build: react-native-cli-tool-maze-runner
+          image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
+          cache-from:  react-native-cli-tool-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:cli-maze-${BRANCH_NAME}
+      - docker-compose#v3.7.0:
+          push: react-native-cli-tool-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:cli-maze-${BRANCH_NAME}
+
+  #
+  # CLI tests
+  #
+  - label: ':runner: RN 0.60 CLI tests'
+    depends_on: "cli-maze-image"
+    timeout_in_minutes: 20
+    plugins:
+      docker-compose#v3.7.0:
+        run: react-native-cli-tool-maze-runner
+        use-aliases: true
+        command: ["-e", "features/built_app.feature"]
```

Comment:
We might want to add `--retry` here to avoid the known flake

Reply:
Ok, we can consider adding that as and when needed.